gateway/src/api/latest.rs

```diff
@@ -475,6 +476,104 @@ async fn route_project(
         .await
 }
 
+#[instrument(skip_all)]
+async fn get_organizations(
+    State(RouterState { service, .. }): State<RouterState>,
+    User { id, .. }: User,
+) -> Result<AxumJson<Vec<organization::Response>>, Error> {
+    let orgs = service.permit_client.get_organizations(&id).await?;
+
+    Ok(AxumJson(orgs))
+}
+
+#[instrument(skip_all, fields(shuttle.organization.name = %organization_name, shuttle.organization.id = field::Empty))]
+async fn create_organization(
+    State(RouterState { service, .. }): State<RouterState>,
+    CustomErrorPath(organization_name): CustomErrorPath<String>,
```

Comment:
Did you apply max string length check somewhere?

Reply:
Lol, forgot about it. Let me update quick

Reply:
Done